.gitlab/e2e/e2e.yml

```diff
@@ -412,15 +412,10 @@ new-e2e-package-signing-suse-a7-x86_64:
 generate-flakes-finder-pipeline:
   image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
   stage: e2e
-  rules: !reference [.on_deploy_nightly_repo_branch]
+  rules:
+    - !reference [.on_deploy_nightly_repo_branch]
+    - !reference [.manual]
   needs:
-    - deploy_deb_testing-a7_arm64
-    - deploy_deb_testing-a7_x64
-    - deploy_rpm_testing-a7_arm64
-    - deploy_rpm_testing-a7_x64
-    - deploy_suse_rpm_testing_arm64-a7
-    - deploy_suse_rpm_testing_x64-a7
-    - deploy_windows_testing-a7
```

Comment:
I am not sure they are used, need to double check with @KevinFairise2 next week. It looks like we use everywhere the `E2E_PIPELINE_ID`, so I guess they are not required, because we use parents pipeline's artifacts.

Anyway requiring @KevinFairise2 review before merging this

Reply:
Agree with Kylian. If we remove that `needs` the pipeline will be generated before all the artifacts are deployed which will make some tests in the flake finder fails.
After removing agent 6 e2e tests that should work as expected if we keep the needs

Reply:
Is [this](https://github.com/DataDog/datadog-agent/pull/28923/files#diff-704c20edf482d3acce6fa8acd84ddf4e4fb11bde2bf3996298b9b997bb895efaR5) not enough for removing agent 6 e2e tests?

Reply:
Nevermind, you are right! We need those dependencies, but they are not always deployed. Checking with agent-delivery if we can enable them on every dev pipeline